python/kotekan/scripts/convert_baseband.py

```diff
@@ -154,8 +154,13 @@ def convert_data(sqlite, conn, e, num_threads):
     if unlocked is True or datetime.datetime.utcnow() > datetime.datetime.strptime(
         e[1], "%Y-%m-%d %H:%M:%S.%f"
     ) + datetime.timedelta(hours=3):
```

Comment:
Why is this 3 hours?  Didn't we say this should be 6 hours below?

Reply:
yes. Fixed it.